client/src/index.html

```diff
@@ -17,7 +17,9 @@
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet"
     />
-  </head>
+      <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
```

Comment:
remove this link

Reply:
both the links are different,someone else has added it